app/forms/mappings/Constraints.scala

```diff
@@ -29,6 +29,7 @@ trait Constraints {
   val regexSortCode: String = """\d{6,}""".r.toString()
   val regexUtr = """^\d{10}$"""
   val regexName = """^[a-zA-Z\u00C0-\u00FF '‘’\u2014\u2013\u2010\u002d]{1,35}$"""
+  val regexUserResearch = """^[a-zA-Z\u00C0-\u00FF '‘’\u2014\u2013\u2010\u002d]{1,70}$"""
```

Comment:
You should just reference the regexName. Alternatively, if we want to do different regexes then it should be looser as we are not passing it through a schema.